services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/IntentionalRerouteSpec.groovy

```diff
@@ -16,8 +16,10 @@ import org.openkilda.testing.service.northbound.NorthboundService
 import org.openkilda.testing.tools.IslUtils
 
 import org.springframework.beans.factory.annotation.Autowired
+import spock.lang.Narrative
 
 @CleanupSwitches
+@Narrative("Spec verifies that on-demand reroute operations are performed accurately.")
```

Comment:
You use construction 'Verify ...' everywhere, but here you use 'Spec verifies ...'. It is not a big deal but can be improved.

Reply:
Fixed, but I don't want to have too strict rules for a 'Narrative'. This should be literally a *narrative* format to describe the purpose

Reply:
Yes, I am fine with it. I just gave a recommendation :)